src/it/MPMD-280-JDK12/invoker.properties

```diff
@@ -16,4 +16,5 @@
 # under the License.
 
 invoker.java.version = 12+
```

Comment:
But here 12?

Reply:
that's correct - these are different test cases:

* src/it/MPMD-270-325-JDK11/pom.xml verifies java11 usage
* src/it/MPMD-280-JDK12/invoker.properties verifies java12 usage